### PR TITLE
Clarify that multiple property values may be of different types

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes Issue 538.

### CODE REVIEW

**Comment by gkellogg** [comment]
@backbone87 see if this helps clarify the language.

File: spec/latest/json-ld/index.html [hunk 1]
```diff
@@ -2466,7 +2537,7 @@ <h3>Graph Containers</h3>
       creating an <a>implicitly named graph</a>. When expanded, these become
       <a>simple graph objects</a>.</p>
 
- <p>An alternative to our example above could use an anoymously named graph
+ <p>An alternative to our example above could use an anoymously <a>named graph</a>
```
Comment (dlongley) [inline spec/latest/json-ld/index.html]: typo: anoymously => anonymously

**Comment by backbone87** [comment]
Regarding #538 : very clear now

**Comment by gkellogg** [comment]
We used "named graph" throughout the document, as that is the data model construct we're drawing upon. A more pervasive change might be to create the term "subgraph", and define it in terms of the RDF "named graph".

Similar is the use of "graph name", vs graph label, and the fact that even unlabeled subgraphs logically still have a name/label.

However, this is was cleaned up text and the issue/PR is about clarifying multiple heterogeneous values of a property, and not really about named graphs. A new issue about creating an abstraction level above "named graphs" is probably a better way to go.

**Comment by gkellogg** [comment]
I went ahead and did some improvement of unnamed named graphs. Anything else is a big change, which should be discussed. Personally, I think it's use among non-expert users is limited, and an understanding of the underlying RDF data model is important in such cases.

**Pull request merged**